src/main/scala/dpla/ingestion3/reports/PrepareEnrichmentReport.scala

```diff
@@ -0,0 +1,87 @@
+package dpla.ingestion3.reports
+
+import dpla.ingestion3.messages.{IngestMessage, IngestMessageTemplates, MessageCollector}
+import dpla.ingestion3.model.OreAggregation
+import dpla.ingestion3.reports.summary.ReportFormattingUtils
+import dpla.ingestion3.utils.Utils
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.{Dataset, Row}
+
+object PrepareEnrichmentReport extends IngestMessageTemplates {
+
+  def generateFieldReport(ds: Dataset[Row], field: String) = {
+    val queryResult = ds
+      .select("level", "message", "id", "field")
+      .where(s"field=='$field'")
+      .drop("field")
+      .groupBy("level", "message")
+      .agg(count("id")).as("count")
+      .orderBy("level","message")
+      .orderBy(desc("count(id)"))
```

Comment:
Could this be one `orderBy` statement instead of two?  Not a big deal, mostly I'm just curious.

Reply:
I thought it complained about something when I put it on a single line but I might be misremembering.